arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,757 @@
+        "defaultServiceTypeDeltaHealthPolicy": {
+          "$ref": "#/definitions/PaasServiceTypeDeltaHealthPolicy"
+        },
+        "serivceTypeDeltaHealthPolicies": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/PaasServiceTypeDeltaHealthPolicy"
+          }
+        }
+      }
+    },
+    "PaasClusterUpgradeDeltaHealthPolicy": {
+      "description": "Paas cluster upgrade delta health policy",
+      "properties": {
+        "maxPercentDeltaUnhealthyNodes": {
+          "type": "integer"
+        },
+        "maxPercentUpgradeDomainDeltaUnhealthyNodes": {
+          "type": "integer"
+        },
+        "maxPercentDeltaUnhealthyApplications": {
+          "type": "integer"
+        },
+        "applicationHealthPolicies": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/PaasApplicationDeltaHealthPolicy"
+          }
+        }
+      }
+    },
+    "paasClusterUpgradePolicy": {
+      "description": "Paas cluster upgrade policy",
+      "properties": {
+        "overrideUserUpgradePolicy": {
+          "type": "boolean"
+        },
+        "forceRestart": {
+          "type": "boolean"
+        },
+        "upgradeReplicaSetCheckTimeout": {
+          "type": "string"
+        },
+        "healthCheckWaitDuration": {
+          "type": "string"
+        },
+        "healthCheckStableDuration": {
+          "type": "string"
+        },
+        "healthCheckRetryTimeout": {
+          "type": "string"
+        },
+        "upgradeTimeout": {
+          "type": "string"
+        },
+        "upgradeDomainTimeout": {
+          "type": "string"
+        },
+        "healthPolicy": {
+          "$ref": "#/definitions/PaasClusterUpgradeHealthPolicy"
+        },
+        "deltaHealthPolicy": {
+          "$ref": "#/definitions/PaasClusterUpgradeDeltaHealthPolicy"
+        }
+      }
+    },
+    "diagnosticsStorageAccountConfig": {
+      "description": "Diagnostics storage account config",
+      "properties": {
+        "storageAccountName": {
+          "type": "string"
+        },
+        "primaryAccessKey": {
+          "type": "string"
+        },
+        "secondaryAccessKey": {
+          "type": "string"
+        },
+        "protectedAccountKeyName": {
+          "type": "string"
+        },
+        "blobEndpoint": {
+          "type": "string"
+        },
+        "queueEndpoint": {
+          "type": "string"
+        },
+        "tableEndpoint": {
+          "type": "string"
+        }
+      }
+    },
+    "clusterResourceProperties": {
+      "description": "The cluster resource properties",
+      "properties": {
+        "availableClusterVersions": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ClusterVersionDetails"
+          }
+        },
+        "clusterId": {
+          "type": "string"
+        },
+        "clusterState": {
```

Comment:
"clusterState" is a property name, should we make the first letter as lower case ?